src/modules/commentCount.js

```diff
@@ -0,0 +1,11 @@
+/* eslint import/prefer-default-export: "off" */
+import { commentUrl } from './api.js';
+
+const testComments = async (id) => {
+  const commenturl = `?item_id=${id}`;
+  const response = await fetch(commentUrl + commenturl);
+  const result = await response.json();
+  return result;
+};
```

Comment:
- Kindly make sure to follow the guidelines in the card `Add comments counter - Student B` in this [issues](https://github.com/zuhebahmed88091/ShowAddit_capstone_js/issues/5): 
   - [ ] A counter function should look for specific DOM elements. 
   - [ ] A counter function should cover all the edge cases you can think about.